README.md

```diff
@@ -181,7 +181,7 @@ latest version.
 
 ## Uninstalling Julia
 
-Julia does not install anything outside the directory it was cloned into. Julia can be completely uninstalled by deleting this directory. Julia packages are installed in `~/.julia` by default, and can be uninstalled by deleting `~/.julia`.
+Julia does not install anything outside the directory it was cloned into. Julia can be completely uninstalled by deleting this directory. Julia packages are installed in `~/.julia` (`~/Library/Julia` on macOS) by default, and can be uninstalled by deleting `~/.julia` (`~/Library/Julia` on macOS).
```

Comment:
I chose a top-level Library directory to match Apple's own uses (see `[~]/Library/{Perl,Ruby,Python}`.

The App Support sub-directory would be appropriate for files specific to the repl/ui (e.g. `Library/Application Support/Julia/.julia_history`, although I doubt anyone would want to move `.julia_history` out of `$HOME`)